rpc/src/endpoint/status.rs

```diff
@@ -41,6 +41,20 @@ impl crate::Response for Response {}
 /// Sync information
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct SyncInfo {
+    /// Earliest block hash
```

Comment:
Now only the `catching_up` field is not represented in this structure.
Maybe we should add it as well.

Reply:
Catching up is already there (end of SyncInfo struct) 👍